### DISCUSSION TITLE
FAQ für Wärmepumpen Setup ?

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#465] [Q&A]
Hallo es wäre doch eine tolle Idee eine FAQ für das Wärmepumpen setup zu erstellen was meint ihr ?

zb.

1. Für Wp mit SG-Rady müssen 2 Verbrauchen im SAE angelgt werden. Beim Verbraucher 1 muss  Slave Schalter muss eingeschaltet sein.

usw.

**Comment by camueller** [comment]
Steht alles schon hier: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/SwitchOnOption_DE.md
